mkdocs_static_i18n/plugin.py

```diff
@@ -322,6 +344,21 @@ def on_config(self, config, **kwargs):
                                 "lang": language,
                             }
                         )
+        # Support for the search plugin lang
+        if "search" in config["plugins"]:
+            search_langs = config["plugins"]["search"].config["lang"] or []
+            for language in self.all_languages:
+                if language in LUNR_LANGUAGES:
+                    if language not in search_langs:
+                        search_langs.append(language)
+                        log.info(
+                            f"Adding '{language}' to the 'plugins.search.lang' option"
+                        )
+                else:
+                    log.warning(
+                        f"Language '{language}' is not supported by "
+                        f"lunr.js, not setting it in the 'plugins.search.lang' option"
```

Comment:
Just one question: what is suppose to happens if a language is not supported? I'm testing it with `gl` (Galician language) and I don't see any difference with the other supported language that I have (`en` and `fr`). I mean, I can search for content of my Galician documentation.

Just FYI `gl` is supported by [mkdocs material](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language)

Reply:
It's just about how `lunr` will be smart about parsing this language [according to the lunr documentation](https://github.com/MihaiValentin/lunr-languages#indexing-multi-language-content). So it works yes, maybe not so optimally for certain sentences / words depending on the language.

Reply:
Great! Thanks for the explanation 😃